### PR TITLE
Upgrade jinja to fix CVE

### PR DESCRIPTION
**Pull request by fhcarter**
Upgrade jinja for dependabot inspired fix for CVE.

Also fix cluster count for staging-based tests

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -101,10 +101,6 @@ pytest-metadata==3.0.0
     # via pytest-html
 pytest-timeout==2.2.0
     # via -r requirements-build.in
-pywin32-ctypes==0.2.2 ; platform_system == "Windows"
```
Comment (sfitts) [inline requirements.txt:104]: Don't remove this

File: requirements.txt [hunk 2]
```diff
@@ -26,14 +28,10 @@ charset-normalizer==3.3.2
     # via requests
 click==8.1.7
     # via pip-tools
-colorama==0.4.6 ; platform_system == "Windows"
```
Comment (sfitts) [inline requirements.txt:29]: Don't remove this

File: src/test/python/test_VantiqSDKLive.py [hunk 3]
```diff
@@ -603,7 +603,7 @@ async def check_crud_operations(self, client: Vantiq, skip_pretest_cleanup: bool
         assert isinstance(vr, VantiqResponse)
         assert vr.is_success
         rows = vr.body
-        assert len(rows) == 0
+        assert len(rows) == 1
```
Comment (sfitts) [inline src/test/python/test_VantiqSDKLive.py:606]: Did something fix this?

**Review by sfitts — APPROVED** [review]
LGTM

File: src/test/python/test_VantiqSDKLive.py (hunk 3 above)
Comment (fhcarter) [reply src/test/python/test_VantiqSDKLive.py:606]: I think that when I created the tests, the _self_ cluster wasn't around always.  So it didn't show up -- now it does.  That's all.

File: requirements.txt (hunk 2 above)
Comment (fhcarter) [reply requirements.txt:29]: Yeah -- I forgot to go back & fix those up.

**Pull request merged**